app/index.tsx

```diff
@@ -1,17 +1,20 @@
 import { Redirect, Route, Router, Switch } from "wouter"
+import { useHashLocation } from "wouter/use-hash-location"
 import { useShallow } from "zustand/react/shallow"
 
-import { Navbar } from "~app/component/navbar"
 import { ProviderContextProvider } from "~app/context/provider"
+import { UserOperationAuthorization } from "~app/page/authorization/userOperation"
 import { Info } from "~app/page/info"
 import { Send } from "~app/page/send"
+import { WebAuthnAuthentication } from "~app/page/webauthn/authentication"
+import { WebAuthnDevTool } from "~app/page/webauthn/devtool"
+import { WebAuthnRegistration } from "~app/page/webauthn/registration"
 import { Path } from "~app/path"
 import { useStorage } from "~app/storage"
-import { useHashLocation } from "~app/util/location"
 
 import "~style.css"
```

Comment:
Discovery unrelated to the purpose of this PR:
In the past, `webauthn/authentication` had no CSS set.
After adding TailwindCSS v3.0 (TailwindCSS 4 is on the horizon!), because `webauthn/authentication` didn't have a `className` set, it didn't have any formatting.

Reply:
Yes, you're right. We can polish them in the future PR 😎